hr_employee_relative/tests/test_hr_employee_relatives.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo.tests import Form, common
+
+_ns = "hr_employee_relative"
```

Comment:
Is there really a need for this? It's just reused twice and the legibility is much worse in the self.env.ref
In fact you could load at the beginning of the test self.env.ref("hr_employee_relative.relation_sibling") and reuse it in lines 28 and 48